### ISSUE TITLE
Memos is not reading from server

### ISSUE THREAD

**Issue by haripalat** [mudkipme/MoeMemos#212]
Something broke with the sync. Can you fix it or help me rest it? 
I deleted the app, removed iCloud backup and tried again and still nothing.

**Comment by haripalat** [comment]
Any update ? I am not able to use the app anymore.

**Comment by haripalat** [comment]
Hi any update ? I actually loved the app when it worked. Now it isn’t working

**Comment by lirc572** [comment]
+1 the app is not able to read memos from the server running latest version 0.22.5.

**Comment by SirSharanChakradhar** [comment]
+1

**Comment by snibles** [comment]
Same situation... on iOS, I can create new memos, but nothing is read from server. On Android, everything is working. 
I'm running iOS 18.1.

**Comment by mudkipme** [comment]
Are there any irregular server logs when experiencing this issue? Does the Explore/Archived or tag pages work?

**Comment by lirc572** [comment]
> Are there any irregular server logs when experiencing this issue? Does the Explore/Archived or tag pages work?

Can see tags in the app, but no memo listed under each tag.

The explore tab shows memos from other accounts, but the logged-in user's memos are not shown anywhere.

In server log, everything is `OK`.

**Comment by snibles** [comment]
> Are there any irregular server logs when experiencing this issue? Does the Explore/Archived or tag pages work?

Not that I can see.

Explore and Archive pages are showing as blank.
Interestingly it can load my tags, but when I click on them, nothing shows up.

![photo_2024-10-31_10-17-28](https://github.com/user-attachments/assets/8795ba51-5913-4fc8-90be-dc7d46eb2475)
![photo_2024-10-31_10-17-33](https://github.com/user-attachments/assets/fda9daf6-db17-490f-91e5-a3909514d949)
![photo_2024-10-31_10-17-21](https://github.com/user-attachments/assets/e5beddff-fb12-40b5-ae8e-bc7e55afe4a8)

It only shows the note created now on the iPhone.

![photo_2024-10-31_10-16-58](https://github.com/user-attachments/assets/4743c9d2-9261-4c74-b0cb-01244ed10543)

**Comment by mudkipme** [comment]
To find out whether this is caused by the content of memos or server environment, could you please create a new user on the same server, and save a simple memo and check whether this issue exists with the new user?

**Comment by haripalat** [comment]
I am using stable and same issue as explain above.

**Comment by snibles** [comment]
> To find out whether this is caused by the content of memos or server environment, could you please create a new user on the same server, and save a simple memo and check whether this issue exists with the new user?
> 
> If you are using the pre-release versions of Memos (the `latest` docker tag), please try backup the data and switch to `stable` or `0.22.5` release.

I´m using Memos stable 0.22.5
I created a new user, and same situation happens on the new user.

New note created on iPhone is visible, but note created on the web is not seen on mobile.

**Comment by lirc572** [comment]
> New note created on iPhone is visible, but note created on the web is not seen on mobile.

Memos created from the app are visible at first but disappear after a reload. Maybe it's cached when created, but retrieval from the server does not work no matter how the memos are created.

**Comment by snibles** [comment]
> > New note created on iPhone is visible, but note created on the web is not seen on mobile.
> 
> Memos created from the app are visible at first but disappear after a reload. Maybe it's cached when created, but retrieval from the server does not work no matter how the memos are created.

You're right. I didn't noticed that.
After I closed the mobile app, the note disappeared. Blank screen.
However the note is still present on server and I can see it on the browser.

**Comment by haripalat** [comment]
To add to the issue. I can create a note in mobile and it shows up in web. If I make a change in web. It does not show in mobile app. Also if I make a change in mobile, it will just override the web version and web version changes are lost as the sync in broken.

**Comment by snibles** [comment]
Update:
So, I deleted my docker container for Memos and redeployed.
The container was updated to version 0.23.0 as shown on image.

![Screenshot from 2024-10-31 14-52-29](https://github.com/user-attachments/assets/a41ce442-86ca-49ae-ae33-c967057579d8)

Now the application on iOS is showing all my notes on the server. New notes when created show an error, but are synced to server. I cannot get the full error message...

![photo_2024-10-31_14-58-36](https://github.com/user-attachments/assets/b33cdfc8-8c6b-447d-9876-5309874b252d)
